src/cidr/ipv4_cidr.rs

```diff
@@ -218,7 +218,7 @@ impl Ipv4Cidr {
     }
 }
 
-#[derive(Debug, PartialEq)]
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Copy, Clone, Hash)]
```

Comment:
I disagree with implementing Copy, Eq, PartialOrd, Ord, and Hash on the error types.